tests/amd_migraphx_tests/mnist/migraphx_node_test.cpp

```diff
@@ -1,3 +1,25 @@
+/*
```

Comment:
@kiritigowda The amd_openvx_extension migrpahx reads copyright 2021-, here it's 2019-
Do we want to make it consistent

Reply:
Similarly other tests for migraphx. 
Tests under inference also have mismatching start years in case you want to make them consistent